README.md

```diff
@@ -13,88 +16,45 @@ npm i -g good-first-issue
 good-first-issue <project>
 ```
 
+### As Module
+
+```js
+const gfi = require('good-first-issue')
```

Comment:
I personally prefer to have more comprehensive variable names, especially in docs. Perhaps:

```suggestion
const goodFirstIssue = require('good-first-issue')
```